metricproviders/prometheus/prometheus.go

```diff
@@ -183,7 +185,17 @@ func NewPrometheusAPI(metric v1alpha1.Metric) (v1.API, error) {
 
 	prometheusApiConfig := api.Config{
 		Address: metric.Provider.Prometheus.Address,
+		RoundTripper: &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}).DialContext,
+			TLSHandshakeTimeout: 10 * time.Second,
```

Comment:
What are these defaults, should we make sure to maintain the defaults if we are not exposing them as config?